### ISSUE TITLE
kotlinx-datetime api can not Recognized by idea in wasm environment

### ISSUE THREAD

**Issue by ericCHensssssssss** [Kotlin/kotlinx-datetime#462]
i develop wasm project，If i import kotlinx.datetime. api  in idea develop  Environment.   idea tells me that the api does not exist（it worked well at jvm environment）

**Comment by ericCHensssssssss** [comment]
> i develop wasm project，If i import kotlinx.datetime. api in idea develop Environment. idea tells me that the api does not exist（it worked well at jvm environment）

**Comment by ericCHensssssssss** [comment]
> i develop wasm project，If i import kotlinx.datetime. api in idea develop Environment. idea tells me that the api does not exist（it worked well at jvm environment）

This is the screen shot ：

**Comment by ericCHensssssssss** [comment]
I  build on my local machine . and can find the date for wasm/WASI is success import by gradle

**Comment by dkhalanskyjb** [comment]
The commit https://github.com/Kotlin/kotlinx-datetime/commit/30ab530c3ee41f43282fdaaa77e814df2f952235, which introduces Wasm/WASI support to this library, was not in the version 0.6.1, and we didn't publish any version after that. Are you building `kotlinx-datetime` directly from `master`?

In any case, if the code successfully builds, but IDEA reports it as red, please report it to https://youtrack.jetbrains.com/, the KTIJ subproject.

**Comment by ericCHensssssssss** [comment]
yes .  i  build kotlinx-datetime directly from master.(The latest codes pulled)  , but IDEA still reports it as red.

**Comment by ericCHensssssssss** [comment]
And when will you publish the version support Wasm/WASI

**Comment by dkhalanskyjb** [comment]
Since the code compiles, but the IDE incorrectly says that there are errors in the code, this means there is a problem with the IDE, so it needs to be reported to https://youtrack.jetbrains.com/newissue?project=KTIJ
